tests/model/component_import.cpp

```diff
@@ -201,14 +188,14 @@ TEST(ComponentImport, multipleImportAndParse)
 
     libcellml::PrinterPtr printer = libcellml::Printer::create();
     std::string a = printer->printModel(m);
-    EXPECT_TRUE((e1 == a) || (e2 == a));
+    EXPECT_EQ(e, a);
```

Comment:
This is wrong, `c3` should be a in a separate import to `c1` and `c2`.

Reply:
Because it was created with separate ImportSource pointers? Or is there another way to tell?

Reply:
yep, that is the only way to tell I think

Reply:
Agreed and that means that we will somehow need to keep track of that fact when parsing a CellML file.

Reply:
OK, I'll take out the check-for-duplicate-url bit, easy enough.

Reply:
Done in fca5344